cmd/install/install.go

```diff
@@ -65,154 +62,54 @@ var Cmd = &cobra.Command{
 		}
 
 		if !config.HasPassword() {
-			err = askForPassword()
+			err = credentials.AskForPassword()
 			if err != nil {
 				log.Panicf("Could not get a password: %v", err)
 			}
 		}
 
-		if !viper.IsSet(config.WwiseEmail_key) {
-			err = askForWwiseAuth()
-			if err != nil {
-				log.Panicf("Could not log in with Wwise: %v", err)
-			}
-		}
-
-		wwiseEmail, err := config.GetSecretString(config.WwiseEmail_key)
+		wwiseCredentials, err := credentials.GetWwiseCredentials()
 		if err != nil {
-			log.Panicf("Could not get the Wwise email: %v", err)
-		}
-
-		wwisePassword, err := config.GetSecretString(config.WwisePassword_key)
-		if err != nil {
-			log.Panicf("Could not get the Wwise password: %v", err)
-		}
-
-		_, err = gh.GetToken()
```

Comment:
The early GH token get was to make sure the user didn't need to interact with the program once the initial back and forth was done and it was intended to check if the credentials were needed before actually getting them, later on. For now this is fine but we should remember to do that check and make sure we do have all the interactivity at the beginning

Reply:
Thanks for the info. I am pretty sure this is still the case because the UE installer is the first install step, and wwise credentials are still collected in advance of that step. If you think this is sufficient guarantee, please resolve the conversation since I've read it now.

Reply:
See 9bb11e7